docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -35,78 +38,85 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 
 .. tab:: Software
 
-  * For C++ developers:
-     * `Microsoft Visual Studio 2019 with MSBuild <http://visualstudio.microsoft.com/downloads/>`_
-     * `CMake 3.14 or higher, 64-bit <https://cmake.org/download/>`_ (optional, only required for building sample applications)
-  * For Python developers: `Python 3.6 - 3.10, 64-bit <https://www.python.org/downloads/windows/>`_
-     * Note that OpenVINO is gradually stopping the support for Python 3.6. You are encouraged to use Python 3.7 - 3.10.
+  * `Microsoft Visual Studio 2019 with MSBuild <http://visualstudio.microsoft.com/downloads/>`_
```

Comment:
We can add something like "MSVC of version 2019 at least", because 2022 is also supported